src/community/geogig/src/main/java/org/geogig/geoserver/config/GeoServerGeoGigRepositoryResolver.java

```diff
@@ -62,7 +74,7 @@ public void initialize(URI repoURI, Context repoContext) throws IllegalArgumentE
         throw new UnsupportedOperationException("Not supported yet.");
     }
 
-    @Override
+
     public ConfigDatabase getConfigDatabase(URI repoURI, Context repoContext, boolean rootUri) {
```

Comment:
If this is overriding a method from an interface, it should still use `@Override`
If it isn't, then why does this method even exist, if it is not supported?